hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -2311,6 +2311,51 @@ message DeleteObjectTaggingRequest {
 message DeleteObjectTaggingResponse {
 }
 
+/**
+S3 lifecycles (filter, expiration, rule and configuration).
+ */
+message LifecycleFilterTag {
+  required string key = 1;
+  required string value = 2;
+}
+
+message LifecycleRuleAndOperator {
+  optional string prefix = 1;
+  repeated LifecycleFilterTag tags = 2;
+}
+
+message LifecycleFilter {
+  optional string prefix = 1;
```

Comment:
Please add the comment about oneof relationship and proto2 limitation for 
1. LifecycleFilter
2. LifecycleExpiration
3. LifecycleRule prefix and filter.

Reply:
Added